marketplace_builder/views/pages/search.liquid

```diff
@@ -0,0 +1,24 @@
+---
+slug: 'search'
+metadata:
+  breadcrumbs: [ { name: 'Search', path: '/search' } ]
```

Comment:
is there any other way of doing it that wont require manually filling this thing on every page?
ie. using `slug titleized` + path `../` or whatever? i can see this being huge maintenance burden in the nearest future